infrap4d/CMakeLists.txt

```diff
@@ -5,28 +5,37 @@ project(infrap4d VERSION 0.1.0 LANGUAGES C CXX)
 set(STRATUM_TDI_BIN_DIR ${STRATUM_SOURCE_DIR}/stratum/hal/bin/tdi)
 set(STRATUM_BFRT_BIN_DIR ${STRATUM_SOURCE_DIR}/stratum/hal/bin/barefoot)
 
-if(DPDK_TARGET)
+if(WITH_KRNLMON)
+    add_executable(infrap4d infrap4d_main.cc)
```

Comment:
Where is `infrap4d_main.cc`? I don't see it in `infrap4d`, and it's not part of this commit.